multimodal/src/autogluon/multimodal/data/process_text.py

```diff
@@ -235,7 +235,7 @@ def collate_fn(self, text_column_names: Optional[List] = None) -> Dict:
 
     def build_one_token_sequence(
         self,
-        text_tokens: Dict[str, NDArray[(Any,), np.int32]],
+        text_tokens: Dict[str, NDArray],
```

Comment:
@zhiqiangdon @cheungdaven I changed the typehints to be just `NDArray` due to the API change of the latest nptyping.

Reply:
`NDArray ` should be fine, but the shape and dtype info are no longer available.